kong/router/transform.lua

```diff
@@ -258,6 +259,44 @@ local function gen_for_nets(ip_field, port_field, vals)
 end
 
 
+local function gen_for_snis(snis)
```

Comment:
We should not create this new function, instead we should use `gen_for_field` with param `sni_op_transform` and `sni_val_transform`

Reply:
applied